### DISCUSSION TITLE
Import error

### DISCUSSION THREAD

**Discussion by zhanming-Chinese** [nonebot/discussions#342] [问答] [answered]
![image](https://user-images.githubusercontent.com/62282155/226151298-a85d307e-1e7e-4a02-a420-7ec636f4eb8e.png)
我在运行我的nonebot时出现如下报错：ImportError: cannot import name 'NotaBene' from partially initialized module 'nb' (most likely due to a circular import) (C:\Users\Administrator\AppData\Local\Programs\Python\Python310\lib\site-packages\nb\__init__.py)，我已确保该库已导入，请告诉我该怎么做。

**Comment by yanyongyu** [comment, marked as answer]
你装错库了啊:joy: 是 `nb-cli` 不是 `nb`

**Reply by Tosd0** [reply]
我靠，，笑死我了我今天排了一堆错，没想到是懒导致的
